### DISCUSSION TITLE
Error dalam Interpretasi Menggunakan LIME

### DISCUSSION THREAD

**Discussion by brigittacarmen** [teamalgoritma/community#748] [Q&A]
Dear Team Algoritma, 

Saya memiliki kesulitan dalam menejermahkan error yang saya dapatkan. Juga masih bingung untuk menggunakan LIME nya.

<img width="470" alt="image" src="https://github.com/teamalgoritma/community/assets/134923285/df91cccd-a26e-4628-b3a1-02ac1415f7aa">

Sementara model Naive Bayes saya sudah berhasil, dan apabila saya gunakan as_classifier juga tidak memberikan pengaruh dalam outputnya.

Any response will be highly appreciated:D
Terima kasih!

Brigitta Carmen

**Comment by amaliasf** [comment]
Dear @brigittacarmen 

Berdasarkan code yang dilampirkan sepertinya sudah tepat Bu. Mungkin agar kami lebih mudah tracing error dan menemukan solusi yang tepat bisa dilampirkan file-nya Bu Brigitaa.

**Comment by kinantytasyao** [comment]
Dear @brigittacarmen 

Mohon maaf untuk menjaga keefektifan diskusi ini yang sudah lebih dari sebulan, maka kami izin untuk menutup diskusi ini ya. Terima kasih sudah bertanya :)
